### PR TITLE
AB facs: x axis labels

### LINKED ISSUE #591: AB: FACS allow user to change axes labels
As an instructor I would like to set the labels and ranges for the x and y axes in my histogram. 

Since the labels and ranges need to be the same for each analysis and condition, the UI should not be on the drawing interface -- which would suggest that they can be set per graph.

This should be a step after 'Sample Prep' and before adding histograms (currently 'Analyze'). The user should be able to set the minimum, maximum and any intermediate values for the x and y axes, and the text labels for each. There should also be a preview function so the instructor can see what graph labels will look like before continuing. 

The instructor will also have to choose whether she wants a linear or logarithmic axis. This will affect which pre-defined histograms she will see in the select histogram interface #574)

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?

Fix #591 
#### What's this PR do?

Take range and tick values for x and display it in the sketching tool.
#### How should this be manually tested?

Play with the upper bound on x and the tick values, and see how that is reflected in the sketching canvas.
#### Any background context you want to provide?

The scale is only linear for now.

### CODE REVIEW

**Comment by annagav** [comment]
@giocalitri, this is rebased and ready for review.

File: instructor/common.py [hunk 1]
```diff
@@ -1471,6 +1471,9 @@ def facs_analyze(request):
     """
     pk = request.session['assignment_id']
     assignment = models.Assignment.objects.get(id=pk)
+    facs, created = models.FlowCytometry.objects.get_or_create(
```
Comment (giocalitri) [inline instructor/common.py]: are you using the variable `created`?

Comment (annagav) [inline instructor/common.py]: not really

File: instructor/ui/sketch_tool.js [hunk 2]
```diff
@@ -38,6 +43,21 @@ function load_sketch_tool() {
         visible: false
     });
 
+    /* Calculating axis labels */
+    tick_values=tick_values.split(",").map(function(x){
+        return parseInt(x)
+    });
+    x_upper_bound = parseInt(x_upper_bound);
+    _.each(tick_values, function(value, index){
+        var x_coor = X_AXIS_LENGTH_PX*value/x_upper_bound;
```
Comment (giocalitri) [inline instructor/ui/sketch_tool.js]: nit: please put some spaces around the operators

File: instructor/common.py (hunk 1 above)
Comment (giocalitri) [inline instructor/common.py]: well, you should write something like this then:

```
facs, _ = models.FlowCytometry.objects.get_or_create
```

unless you use `_` for something else

**Comment by annagav** [comment]
@giocalitri, comments addressed.

**Comment by giocalitri** [comment]
:+1:  after squash

**Pull request merged**